src/services/Store/store/account.slice.ts

```diff
@@ -75,7 +76,19 @@ export default slice;
 /**
  * Selectors
  */
-export const getAccounts = createSelector([getAppState], (s) => s[slice.name]);
+export const getAccounts = createSelector([getAppState], (s) => {
+  const accounts = s[slice.name];
+  return accounts.map((a) => ({
+    ...a,
+    transactions: a.transactions.map((t) => ({
+      ...t,
+      value: EthersBN.from(t.value),
+      gasLimit: EthersBN.from(t.gasLimit),
+      gasPrice: EthersBN.from(t.gasPrice),
+      gasUsed: t.gasUsed && EthersBN.from(t.gasUsed)
```

Comment:
With this, it because useful to include a selector test to assert that the value is `bigish`

Reply:
I don't think I understand entirely what you mean

Reply:
It's a request to test the selector to ensure that the value is correctly transformed from a string to a `Bigish`.